Makefile

```
@@ -52,8 +52,19 @@ fmt:
 
 __REWARDS__: ##
 
-create-and-claim-distribution:
-	./contracts/anvil/create-and-claim-distribution.sh
-
-create-operator-directed-distribution-and-claim:
-	./contracts/anvil/create-operator-directed-and-claim-distribution.sh
+SENDER_ADDR=0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266
+
+create-avs-distributions-root:
+	cd contracts && \
+	forge script script/SetupDistributions.s.sol:SetupDistributions --rpc-url http://localhost:8545 \
+		--broadcast --sig "runAVSRewards()" -v --sender "0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266"
+
+create-operator-directed-distributions-root:
+	cd contracts && \
+	forge script script/SetupDistributions.s.sol:SetupDistributions --rpc-url http://localhost:8545 \
```

I think the contract name is not needed here:

```suggestion
	forge script script/SetupDistributions.s.sol --rpc-url http://localhost:8545 \
```

---

Yes, done in [c8b7ad5](https://github.com/Layr-Labs/incredible-squaring-avs-rs/pull/45/commits/c8b7ad5bb6a2114d5e416dd2f9675ed936bdbc67)